inc/saiport.h

```diff
@@ -368,6 +368,17 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_QOS_SCHEDULER_GROUP_LIST,
 
+    /**
+     * @brief The maximum accumulative headroom size of a port in bytes
+     *
+     * This attribute is applicable only for per-port, per-PG headroom model
+     * (which means SAI_BUFFER_POOL_ATTR_XOFF_SIZE is zero)
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
```

Comment:
What should be the value returned by the platforms that do not have such a limit?

Reply:
They can just return 0.

Reply:
Thanks. Please update this in the comments as well so that the NOS/application handles the special case of 0.

Reply:
Done.